src/references/solution.md

```diff
@@ -3,3 +3,11 @@
 ```rust,editable
 {{#include exercise.rs:solution}}
 ```
+
+<details>
+
+- Note that in `normalize` we were able to do `*item /= mag` to modify each
+  element. This is because we're iterating using a reference to an array, which
```

Comment:
Critically, it's an exclusive reference.

Reply:
Ah good point, I updated to call that out explicitly.